Packs/OpenCTI/pack_metadata.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "Feed OpenCTI",
+    "description": "Ingest indicator feeds from OpenCTI.",
```

Comment:
Let's improve this description (we are not ingesting a feed anymore). Something evolving the creation and manipulation of indicators